src/main/java/de/doubleslash/usb_led_matrix/view/ConfigurationView.java

```diff
@@ -139,7 +141,7 @@ public class ConfigurationView {
             LOG.debug("Reconnection check was interrupted.", e);
             return;
          }
-         turnOffAutomaticallyIfNeeded(LocalTime.now());
+         turnOffAutomaticallyIfNeeded(LocalTime.now(),LocalDate.now());
```

Comment:
does not look like the code was formatted correctly

Reply:
Done.